lib/action_policy/behaviours/scoping.rb

```diff
@@ -11,8 +11,6 @@ module Scoping
       #   - secondly, try to infer policy class from `target` (non-raising lookup)
       #   - use `implicit_authorization_target` if none of the above works.
       def authorized_scope(target, type: nil, as: :default, scope_options: nil, **options)
-        options[:context] && (options[:context] = authorization_context.merge(options[:context]))
-
```

Comment:
I think this line was already useless since `context` will be set by `policy_for`